pages/notifications.js

```diff
@@ -148,10 +167,19 @@ const Page = withT(({ url: { query, query: { context, token } }, t }) => {
               )), () => ' – ')}
             </P>
           )}
+          {!hasCurtain && buttonLinks.length > 0 && buttonLinks.map((link, i) => (
```

Comment:
Looks like buttonLinks is guaranteed to be an array, so you wouldn't need the extra check `buttonLinks.length > 0` because an empty array simply woudn't map anything.